test/functional/segwit.py

```diff
@@ -70,7 +70,7 @@ def getutxo(txid):
 
 def find_unspent(node, min_value):
     for utxo in node.listunspent():
-        if utxo['amount'] >= min_value:
+        if utxo['amount'] >= min_value and utxo['spendable']:
             return utxo
 
```

Comment:
Should `throw` for clarity.

Reply:
Yeah, @promag pointed this out already and I said I would do it. Then I ask a question that nobody answered.

Reply:
I'd say either add the new arg or rename the function and add the missing throw.